yarn-project/txe/src/txe_session.ts

```diff
@@ -89,7 +91,10 @@ type MethodNames<T> = {
  * The name of an oracle function that TXE supports, which are a combination of PXE oracles, non-transpiled AVM opcodes,
  * and custom TXE oracles.
  */
-export type TXEOracleFunctionName = MethodNames<RPCTranslator>;
+export type TXEOracleFunctionName = Exclude<
+  MethodNames<RPCTranslator>,
+  'constructor' | 'handlerAsMisc' | 'handlerAsUtility' | 'handlerAsPrivate' | 'handlerAsAvm' | 'handlerAsTxe'
+>;
```

Comment:
This is an unrelated change to the rest of the PR. It has nothing to do with the zod validation check. Just felt like fixing it here given that I was touching this.